app.py

```diff
@@ -168,6 +236,22 @@ def main():
         except Exception as e:
             _LOGGER.exception(f"An error occurred pushing the metrics: {str(e)}")
 
+    with _METRIC_RUNTIME.time():
+        graph_refresh_package_analyzer()
+
+    if _THOTH_METRICS_PUSHGATEWAY_URL:
+        try:
+            _LOGGER.debug(
+                f"Submitting metrics to Prometheus pushgateway {_THOTH_METRICS_PUSHGATEWAY_URL}"
+            )
+            push_to_gateway(
+                _THOTH_METRICS_PUSHGATEWAY_URL,
+                job="graph-refresh-package-analyzer",
```

Comment:
Just copied and changed to `graph-refresh-package-analyzer`.

Reply:
No need to copy this, metrics will be submitted correctly once.